usaspending_api/etl/transaction_loaders/derived_field_functions_fpds.py

```diff
@@ -50,46 +50,42 @@ def legal_entity_zip5(broker_input):
         match = ZIP_CODE_PATTERN.match(broker_input["legal_entity_zip4"])
         if match:
             return match.group(1)
-    return ""
+    return None
 
 
 def place_of_performance_zip5(broker_input):
```

Comment:
This function no longer appears necessary

Reply:
Given how much we are going to be iterating on this code, I was hoping to just leave some scaffolding in place then follow up with a quick cleanup PR once QA was done.

Reply:
It is better to cleanup code each time it is touched. Waiting until an undetermined future event it a good recipe to neglect the work.